libafl_bolts/src/lib.rs

```diff
@@ -316,6 +316,9 @@ pub enum Error {
     Unsupported(String, ErrorBacktrace),
     /// Shutting down, not really an error.
     ShuttingDown,
+    /// OS error from `std::io::Error::last_os_error`;
+    #[cfg(feature = "std")]
+    OSError(io::Error, String, ErrorBacktrace),
```

Comment:
Should be `OsError` I think (lowercase s)

Reply:
Opened #1944 for this

Reply:
> Should be `OsError` I think (lowercase s)

Yes. I forgot about it. The [`std::ffi::OsString`](https://doc.rust-lang.org/std/ffi/struct.OsString.html) is also named in this style.